dodiscover/constraint/config.py

```diff
@@ -16,18 +16,39 @@ def __contains__(cls, item):
         return True
 
 
-class SkeletonMethods(Enum, metaclass=MetaEnum):
-    """Available methods for learning a skeleton from data.
+class ConditioningSetSelection(Enum, metaclass=MetaEnum):
```

Comment:
Will this go to py-indep?

Reply:
No because this is more of a meta-algorithm that is concerned with given the X, Y, size of conditioning set, and current state of the graph, iterate through the possible conditioning sets.

Reply:
py-indep would be more concerned with once you specify X, Y and Z, what is the pvalue of the hypothesis test: `X \perp Y | Z`